docs/contracts/v4/guides/15-single-sided-liquidity.mdx

```diff
@@ -0,0 +1,423 @@
+---
+title: Single-sided Liquidity
+---
+
+In this example we will create a single-side liquidity position a Buy Limit style range order that  swaps token1 for token0 as the market price falls into your defined tick range.
+
+Fetch the current pool price (sqrtPriceX96)
+
+Compute a -5% target price and align it with a usable tick
+
+Calculate the required liquidity using only token1
+
+Approve the correct token and use mint() to create a position
+
+Monitor and fine-tune the amount1Max buffer for slippage tolerance
+
+## Introduction
+
+This guide will cover how single-side liquidity provisioning can be used to execute **Limit Orders** on Uniswap V4 Pools.
+
+
+This guide will **cover**:
+
+1. Understanding Range Orders
+2. Calculating our Tick Range
+3. Creating a single-side liquidity position
+5. Closing the Limit Order
+
+Before working through this guide, consider checking out the Range Orders [concept page](https://docs.uniswap.org/concepts/protocol/range-orders) to understand how Limit orders can be executed with Uniswap v4.
+
+:::info
+This guide is built using the [v4-template repository](https://github.com/uniswapfoundation/v4-template) and utilises the Config and Constants files as imports for contract addresses.
+:::
+
+## Understanding Range Orders
+
+If you have read the [Range Order Concept page](../../../../concepts/protocol/range-orders.md), you can skip this section.
+
+Positions on a v4 Pool are always created with a Tick range in which their liquidity is accessible to swaps on the Pool.
```

Comment:
this sentence reads really weird / is long to me. i dont think this guide needs to try to explain ticks, range, liquidity, and positions.

instead the intro can reference other pages, and explain why single-sided positions are useful